src/components/Web3Handlers/Web3SuccessHandler.tsx

```diff
@@ -19,5 +20,17 @@ export const Web3SuccessHandler: React.FC<Props> = ({ hash }) => (
         {utils.formatting.shortenAddress(hash)}
       </a>
     </span>
+    <span className="hidden lg:block">
```

Comment:
this is a duplicate of the `<span>` block above but with a line break? why not just add some padding to the message

Reply:
It isn't a duplicate, on big screens we show the whole tx hash and on small screens we truncate the hash

Reply:
i noticed that just before you replied, left a comment [here](https://github.com/horse-link/horse.link/pull/311/files#r1066483725)